index.bs

```diff
@@ -430,6 +432,13 @@ about current and past use of the API.
 Note: this does not imply keeping a log of the actual [=sensor readings=]
 which would have issues of its own.
 
+<h4 id="user-triggered-activation" dfn>User triggered activation</h4>
+
+The information whether the user agent is requested to access [=sensor readings=] due to
+[=triggered by user activation|user triggered activation=], may be used as a
+[=new information about the user's intent|criteria=] to grant access to [=sensor readings|readings=].
```

Comment:
FWIW, I'm uncomfortable saying that user activation conveys intent to grant access to a sensor. It's just way too easy to get a user to activate a page without any indication that the user even particularly interested in the content, let alone that there will be a sensor involved.

However, @ojanvafai has been pushing some interventions that care about user gestures to indicate some minimal involvement with the page. e.g. [beforeunload](https://groups.google.com/a/chromium.org/d/topic/blink-dev/MRWIaGY4Txg/discussion) and [autoplaying audio](https://groups.google.com/a/chromium.org/d/topic/blink-dev/51WbTwn0M_Y/discussion). For low-sensitivity sensors, that might make sense, although I'm having trouble thinking of any that are also too sensitive to give to a page without interaction.